compute/instances/reference-content/understanding-qemu-guest-agent.mdx

```diff
@@ -0,0 +1,112 @@
+---
+meta:
+  title: Understanding the QEMU Guest Agent in Scaleway Instances
+  description: Discover how the QEMU Guest Agent in Scaleway Instances works.
+content:
+  h1: Understanding the QEMU Guest Agent in Scaleway Instances
+  paragraph: Discover how the QEMU Guest Agent in Scaleway Instances works.
+tags: instance qga guemu guest agent
+dates:
+  validation: 2024-10-28
+categories:
+  - compute
+---
+
+Some features of the Instances product require Scaleway's infrastructure to query or exchange information with your Instance. To enable this communication, a software component must run on the guest operating system: the QEMU Guest Agent (QGA).
+
+This document provides essential insights into this mechanism.
+
+<Message type="note">
+  This document does not apply to Instances running the Microsoft Windows operating system.
+</Message>
+
+## What are the features provided by QGA?
+
+Running the QEMU Guest Agent (QGA) on your Instance currently enables the following feature:
+
+- **Automatic network reconfiguration** upon flexible IP attachment or detachment.
+
+Additional features may be added in the future.
+
+## Checking the status of QGA
+
+Since March 1, 2024, all Scaleway-provided GNU/Linux and InstantApp images for Instances come with QGA pre-installed and enabled by default.
```

Comment:
```suggestion
Since March 1st, 2024, all Scaleway-provided GNU/Linux and InstantApp images for Instances come with QGA pre-installed and enabled by default.
```